django_mailcss/templatetags/mailcss.py

```diff
@@ -1,6 +1,11 @@
 from django import template
 
-from django.utils.encoding import smart_text
+try:
+    from django.utils.encoding import smart_str
+except ImportError:
+    # smart_text was replaced by smart_str in django 3.0+ and removed in django 4.0
+    from django.utils.encoding import smart_text as smart_str
```

Comment:
I think this is misleading, why not using just
from django.utils.encoding import smart_text?

Reply:
`smart_text` doesn't exist in django 4.0 and `smart_str` doesn't exist in django before that. I have to choose one name to exist, or set both to `None` when any of them doesn't exist and check in the callsite. I think this is simpler and is the default solution that is used when a code needs to support multiple versions of the same library (or even multiple python versions) with that kind of issue.

What we want is to `from django.utils.encoding import smart_text` only, but if I do that and this gets merged before our migration to at least Django 3.2 happens on capacity-api gets merged, capacity-api will break. And if the migration gets merged without this, then it will break as well.

We can remove change to that simple import once capacity-api has migrated in both stage and prod